src/Modules/UngradedModules/Presentation.jl

```diff
@@ -1,12 +1,16 @@
 ####################
 
 @doc raw"""
-    presentation(M::SubquoModule)
+    presentation(M::SubquoModule; minimal=false)
 
-Return a free presentation of `M`. 
+Return a free presentation of `M`. If `minimal` is set to `true`, the returned
+presentation is minimal.
 """
-function presentation(SQ::SubquoModule)
-  if is_graded(SQ)
+function presentation(SQ::SubquoModule;
+                      minimal=false)
+  if minimal
+    return _presentation_minimal(SQ)
+  elseif is_graded(SQ)
     return _presentation_graded(SQ)
```

Comment:
How do `minimal=true` and a graded module interact? Due to now calling `_presentation_minimal` here, the graded check is skipped. So `_presentation_minimal` would need to special-case graded modules somehow

Reply:
Internally, `_presentation_minimal` uses `prune_with_map` which has to check for gradedness anyway. So `_presentation_minimal` does not have to do any extra checks.

Reply:
Nice, didn't know that